R/add_transition_risk_category_at_company_level.R

```diff
@@ -0,0 +1,49 @@
+add_transition_risk_category_at_company_level <- function(data) {
+  product <- data |>
+    unnest_product()
+
+  risk_categories <- product |>
+    adapt_tr_product_cols_to_tiltIndicator_cols() |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    adapt_tiltIndicator_cols_to_tr_company_cols()
```

Comment:
Looks better. At lease the function doesn't mention column names so keeps the code at a higher level of abstraction. 

A further enhancement may be to move the calls to `adapt*()` outside the body of the function. But that may be an overkill at this stage. 

This idea comes from noticing that `add_*()` functions typically only add columns or rows (e.g. `tibble::add_column()`, `tibble::add_row()`) or -- more closely to what we're doing here -- compute some summary and then add it as a column (e.g. `dplyr::add_count()`). With this approach, the adaptors don't quite belong into the body of the `add_*()` function. Instead we could apply the adaptors to the `data` before/after we call `add_*()`.